Sources/Private/EmbeddedLibraries/EpoxyCore/README.md

```diff
@@ -0,0 +1,39 @@
+## EpoxyCore
```

Comment:
Added some discussion about why we include the source of `EpoxyCore`, plus instructions on how to update this in the future if necessary